browse/templates/archive/single_archive.html

```diff
@@ -0,0 +1,89 @@
+{%- extends "base.html" -%}
+
+{% block head %}
+{{ super() }}
+<link rel="alternate" type="application/rss+xml" title="{{archive['name']}}" href="/rss/{{archive_id}}"/>
+{% endblock head %}
+
+{% block title %}{{archive.name}}{% endblock %}
+
+{% block header_h1 %}
+<h1>
+  > <a href="{{url_for('.list_articles', context=archive_id, subcontext='recent')}}">{{archive_id}}</a>
+</h1>
+{% endblock %}
+
+
+{%- block content %}
+<div id='content'>
+  <h1>{{archive.name}} <span style="font-size: 70%;">(since {{archive.start_date}})</span></h1>
+
+  {% if subsumed_id %}
+    <p>The <b>{{subsumed_id}}</b> archive has been subsumed into
+      <span class="primary-subject">{{subsuming_category['name']}} ({{subsumed_by}}).</span></p>
+    <p>{{subsuming_category['name']}} ({{subsumed_by}}) is part of the {{archive['name']}} archive.</p>
+  {% endif %}
+
+  
+  <p>For a <em>specific paper</em>, enter the identifier into the top right search box.</p>
+
+  <ul>
+    <li><b>Browse:</b>
+      <ul>
+        <li>
+          <a href="{{url_for('.list_articles', context=archive_id, subcontext='new')}}">new</a>
+          (most recent mailing, with abstracts)
+        </li>
+        <li><a href="{{url_for('.list_articles', context=archive_id, subcontext='recent')}}">recent</a>
+          (last 5 mailings)
+        </li>
+        <li><a href="{{url_for('.list_articles', context=archive_id, subcontext='current')}}">current month's</a>
+          nlin listings</li>
+        <li>specific year/month:          
+          <form action="{{url_for('.list_articles')}}" method="get">
+            {{ list_form.archive }}
+            {{ list_form.year}} {{ list_form.month}}
+            {{list_form.submit}}
+          </form>
+        </li>
+      </ul>
+    </li>
+    
+    <li><b>Catch-up:</b> 
+      <form action="TODO" method="post"> 
+        {{ catchup_form.archive }} 
+        Changes since: 
+        {{ catchup_form.day }} 
+        {{ catchup_form.month }} 
+        {{ catchup_form.year }}  
+        abstracts {{ catchup_form.abstracts }}  
```

Comment:
In classic, the "abstracts" text appears after the form field.

Reply:
This should be fixed. The order of the yyyy mm and day too.